File: 01/mario-more/mario.c
#include <stdio.h>

int main(void)
{
   int height;

   //Frage nach einem int Wert für die Höhe der Pyramide
   do
   {
    printf("Enter Height: ");
    scanf("%i", &height);
   } while (height < 1 || height > 8);

   int spaces = height - 1;

   // Pyramide wird erstellt
   for(int i = 0; i < height; i++)
   {

    //Loop um die Spaces für die linksseitige Pyramide zu generieren
    for(int j = 0; j <= spaces; j++)   
    {
        printf(" ");
    }
    
    spaces--;

    // Loop für linksseitige Pyramide
    for(int k = 0; k < i + 1; k++)
    {
        printf("#");
    }

    // Space zwischen beiden Pyramiden
    printf("  "); 

    // Loop für rechtsseitige Pyramide
    for(int m = 0; m < i + 1; m++)
    {
        printf("#");
    }
    printf("\n");
   }
}

File: 06/sentimental-mario-more/mario.py
while True :
    try:
        height = int(input("Enter Height: "))
        if height > 0 and height < 8:
            break
    except ValueError:
        pass

spaces = height - 1

print()

for i in range(height):

    for j in range(spaces):
        print(" ", end="")

    spaces -= 1

    for k in range(i + 1):
        print("#", end="")

    print("  ", end="")

    for m in range(i + 1):
        print("#", end="")

    print()


